packages/angular/src/generators/init/init.ts

```diff
@@ -0,0 +1,153 @@
+import {
+  addDependenciesToPackageJson,
+  readJson,
+  readWorkspaceConfiguration,
+  Tree,
+  updateWorkspaceConfiguration,
+  writeJson,
+  formatFiles,
+  updateJson,
+} from '@nrwl/devkit';
+import { jestInitGenerator } from '@nrwl/jest';
+import { wrapAngularDevkitSchematic } from '@nrwl/devkit/ngcli-adapter';
+import { cypressInitGenerator } from '@nrwl/cypress';
+
+import { Schema } from './schema';
+import {
+  angularVersion,
+  jestPresetAngularVersion,
+  rxjsVersion,
+} from '../../utils/versions';
+import { E2eTestRunner, UnitTestRunner } from '../../utils/test-runners';
+import { setDefaultCollection } from '@nrwl/workspace/src/utilities/set-default-collection';
+
+export default async function (host: Tree, options: Schema) {
+  setDefaults(host, options);
+  addPostInstall(host);
+  updateDependencies(host);
+
+  await addUnitTestRunner(host, options);
+
+  addE2ETestRunner(host, options);
+
+  if (!options.skipFormat) {
+    await formatFiles(host);
+  }
+}
+
+function setDefaults(host: Tree, options: Schema) {
+  const workspace = readWorkspaceConfiguration(host);
+
+  workspace.generators = workspace.generators || {};
+  workspace.generators['@nrwl/angular:application'] = {
+    style: options.style,
+    linter: options.linter,
+    unitTestRunner: options.unitTestRunner,
+    e2eTestRunner: options.e2eTestRunner,
+    ...(workspace.generators['@nrwl/angular:application'] || {}),
+  };
+  workspace.generators['@nrwl/angular:library'] = {
+    style: options.style,
+    linter: options.linter,
+    unitTestRunner: options.unitTestRunner,
+    ...(workspace.generators['@nrwl/angular:library'] || {}),
+  };
+  workspace.generators['@nrwl/angular:component'] = {
+    style: options.style,
+    ...(workspace.generators['@nrwl/angular:component'] || {}),
+  };
+
+  setDefaultCollection(host, '@nrwl/angular');
+  updateWorkspaceConfiguration(host, workspace);
```

Comment:
I'm not sure these 2 operations are in the right order. `setDefaultCollection` will update the workspace configuration in the Tree, but those changes won't be reflected in the `workspace` variable, which you then use to write the configuration. This should effectively remove the changes done to set the default collection.

Reply:
Fixed